projects/plugins/jetpack/_inc/lib/class-jetpack-coauthor-helper.php

```diff
@@ -72,21 +72,43 @@ public static function get_site_id() {
 		return (int) $site_id;
 	}
 
+	/**
+	 * Get the name of the transient for image generation. Unique per prompt and allows for reuse of results for the same prompt across entire WPCOM.
+	 * I expext "puppy" to always be from cache.
+	 *
+	 * @param  string $prompt - Supplied prompt.
+	 */
+	public static function transient_name_for_image_generation( $prompt ) {
+		return 'openai-image-' . md5( $prompt );
+	}
+
+	/**
+	 * Get the name of the transient for text completion. Unique per user, but not per text. Serves more as a cooldown.
+	 */
+	public static function transient_name_for_completion() {
+		return 'openai-completion-' . get_current_user_id(); // Cache for each user, so that other users dont get weird cached version from somebody else.
```

Comment:
What do you think about prefixing those with `jetpack_`, so they're easier to spot for folks using a transient manager and looking for Jetpack transients on their site?

Reply:
Of course!

Reply:
done!